iiwa7:

  ref_velocity:
    x: 0.3
    y: 0.0
    z: 0.0

  ref_orientation:
    w: 0.707
    x: -0.707
    y: 0
    z: 0

  hit_direction:
    x: 0
    y: 1 
    z: 0

  hit_momentum: 3

  return_position:
    x: 0.55
    y: -0.3 # -0.6 offset here IF using gazebo callabck (=gazebo refenrential frame => simulation_referential==True)
    z: 0.4


iiwa14:

  ref_velocity:
    x: 0.3
    y: 0.0
    z: 0.0

  ref_orientation:
    w: 0.707
    x: -0.707
    y: 0
    z: 0

  hit_direction:
    x: 0
    y: -1 
    z: 0

  hit_momentum: 3
    
  return_position:
    x: 0.55 
    y: 0.2  # +0.6 offset here IF using gazebo callabck (=gazebo refenrential frame => simulation_referential==True)
    z: 0.4
    
object_offset:
    iiwa7 :
        x : 0.07
        y : -0.15
        z : -0.05

    iiwa14 :
        x : 0.0
        y : 0.2
        z : 0.0